examples/post_training_quantization/onnx/mobilenet_v2/README.md

```diff
@@ -21,4 +21,28 @@ pip install -r requirements.txt
 It's pretty simple. The example does not require additional preparation. It will do the preparation itself, such as loading the dataset and model, etc.
 ```
 python main.py
-```
\ No newline at end of file
+```
+
+
+# ONNX Results
```

Comment:
In this README I would expect to get only information about the mobilenet_v2 quantization example. It is very strange for me to have here some results for other models. For example, if we add another example, we should put these results to it's README as well?

Reply:
@vshampor

Reply:
Place results in `Readme.md` and `docs/compression_algorithms/post_training/onnx.md`